samples/deploy/crd/v1/ApisixRoute.yaml

```diff
@@ -112,7 +112,18 @@ spec:
                             minItems: 1
                             items:
                               type: string
-                              enum: ["CONNECT", "DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT", "TRACE"]
+                              enum:
+                                [
+                                  "CONNECT",
+                                  "DELETE",
+                                  "GET",
+                                  "HEAD",
+                                  "OPTIONS",
+                                  "PATCH",
+                                  "POST",
+                                  "PUT",
+                                  "TRACE",
+                                ]
```

Comment:
If you want to write a sequence over multiple lines, you can remove the `[` `]`  symbol and use `-`

like:

```
enum:
- "CONNECT"
```